src/dotnet/CarbonAware.Plugins.NewBasicJsonPlugin/CarbonAwareJsonReaderPlugin.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections;
+using CarbonAware.Model;
+using CarbonAware.Plugins.BasicJsonPlugin;
+using Microsoft.Extensions.Logging;
+
+namespace CarbonAware.Plugins.NewBasicJsonPlugin;
+
+public class CarbonAwareJsonReaderPlugin : ICarbonAware
+{
+    private readonly ILogger<CarbonAwareJsonReaderPlugin> Logger;
+    private CarbonAwareStaticJsonDataService CarbonAwareDataService;
+
+
+    public CarbonAwareJsonReaderPlugin(ILogger<CarbonAwareJsonReaderPlugin> logger)
+    {
+        Logger = logger ?? throw new ArgumentNullException(nameof(logger));
+        // CarbonAwareDataService = carbonAwareDataService;
+        CarbonAwareDataService = new CarbonAwareStaticJsonDataService();
+        var file = "/workspaces/carbon-aware-sdk/src/dotnet/data/data-files/sample-emissions-data.json";
+        CarbonAwareDataService.SetFileName(file);
+    }
+
+    public async Task<IEnumerable<EmissionsData>> GetEmissionsDataAsync(Dictionary<string, object> props)
+    {
+        Logger.LogInformation("New Data");
+        foreach (var k in props.Keys)
+        {
+            Logger.LogInformation($"Key: {k} - Value: {props[k]}");
+        }
+        
+        List<string> locations = props.TryGetValue(CarbonAwareConstants.LOCATIONS, out object locs) ? (List<string>)locs : new List<string>();
```

Comment:
@juzuluag we should decide if this is how we want to do this parsing. I left a comment below about why Priti and I changed the IDictionary to Dictionary to enable this because it's a quick solution to ensure things are working. We can decide either now as a squad, or when we put up to PR formally and get rest of teams' opinion